tests/ui/proc-macro/nonfatal-parsing.stderr

```diff
@@ -22,6 +22,24 @@ help: consider inserting whitespace here
 LL | c 'r'
    |  +
 
```

Comment:
Wait, you didn't modify `tests/ui/proc-macro/nonfatal-parsing.rs`, did you? So why did the stderr change (adding of two errors)?

Reply:
because the non-auxiliary was 2024 so I wanted to respect the decision of the person who made the test